frontend/frontend.sh

```diff
@@ -4,12 +4,13 @@ COMPOSE_FILE="docker-compose.yaml"
 
 build_frontend() {
     echo "Building frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE build frontend
+    docker-compose  -f $COMPOSE_FILE build frontend --no-cache
 }
 
 run_frontend() {
     echo "Running frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE --compatibility up -d frontend
+    docker-compose down
+    docker-compose -f $COMPOSE_FILE --compatibility up -d frontend --build
```

Comment:
docker-compose -f $COMPOSE_FILE --compatibility up -d  --build frontend

Reply:
Yes, we will change the order of the command. Unclear why this is needed but we can't find any other way of making it work.